homeassistant/components/tfiac/climate.py

```diff
@@ -29,13 +28,20 @@
 
 MIN_TEMP = 61
 MAX_TEMP = 88
-OPERATION_LIST = {
+OPERATION_MAP = {
     STATE_HEAT: 'heat',
     STATE_AUTO: 'selfFeel',
     STATE_DRY: 'dehumi',
     STATE_FAN_ONLY: 'fan',
     STATE_COOL: 'cool',
 }
+OPERATION_MAP_REV = {
+    'heat': STATE_HEAT,
```

Comment:
```py
OPERATION_MAP_REV = {
    v: k for k, v in OPERATION_MAP.items()}
```

Reply:
I thought you didn't like that (you rejected that in a previous Daikin pr).

Reply:
Why did I do that?

Reply:
I can't find that comment now but it seems like you approved this: https://github.com/home-assistant/home-assistant/pull/16913/files 😄

Reply:
Yeah, sometimes things slip through. :smile: